scripts/upgrade

```diff
@@ -144,31 +93,31 @@ ynh_install_composer
 #=================================================
 # MODIFY A CONFIG FILE
 #=================================================
-#ynh_script_progression --message="Adding a configuration file..." --weight=1
+ynh_script_progression --message="Adding a configuration file..." --weight=1
 
-#app_url_domain="https://$domain${path_url%/}"
+app_url_domain="https://$domain${path%/}"
 
-#ynh_add_config --template=../conf/.env.example --destination=$final_path/.env
+ynh_add_config --template=../conf/.env.example --destination=$install_dir/.env
```

Comment:
I'm confused about why this is uncommented since it's supposed to be "kept" from earlier during `ynh_setup_source`

In package_check we can see that it complains about : 
>Apparently the log is telling that 'some file got manually modified' ... which should not happen, considering that no human modified the file ... ! Maybe you need to check what's happening with ynh_store_file_checksum and ynh_backup_if_checksum_is_different between install and upgrade.

and the `MAIL_PASSWORD` and `APP_KEY` values get overwritten ... 

Though it's never clear wether or not we should upgrade conf file, considering that some day the conf file format might change and we indeed need to "upgrade" it the file ...

But at least it doesn't make much sense to overwrite the MAIL_PASSWORD variable (though I'm confused if the entire `MAIL_*` config actually does something considering the mail password is never actually set on the unix user x_X ?)

Anyway, I dunno exactly but I would just remove that `ynh_add_config` if there's no "real" reason to have it ..

Reply:
As I recall `ynh_add_config` was updated for the addition of `__AUTH_METHOD__` also used in config panel.

Reply:
And I agree, `MAIL_PASSWORD` probably does nothing

Reply:
Actually I think the issue is related with the fact that composer/artisan? (or idk which tool) in the install does overwrite the value of "API_KEY" at some point. Once the checksum changes ... hence we should be calling `ynh_store_file_checksum` (I dont remember the exact name) after this happens ...

But that's not the full story because next time we call `ynh_add_config`, it wont complain, BUT we are still gonna overwrite the value of APP_KEY, which we probably don't want to loose ...

Reply:
Maybe instead of generating the APP_KEY with `php$phpversion artisan key:generate --force` we can set a ramdom key in the install script and reuse it in the upgrade